src/researchhub_document/management/commands/backfill_papers_from_openalex.py

```diff
@@ -0,0 +1,71 @@
+"""
+The purpose of this script is to enrich local papers with OpenAlex data.
+Such data includes topics, subfields, etc...
+"""
+
+from django.contrib.contenttypes.models import ContentType
+from django.core.management.base import BaseCommand
+
+from paper.openalex_util import process_openalex_works
+from paper.related_models.paper_model import Paper
+from researchhub_case.related_models.author_claim_case_model import AuthorClaimCase
+from researchhub_comment.related_models.rh_comment_thread_model import (
+    RhCommentThreadModel,
+)
+from utils.openalex import OpenAlex
+
+
+class Command(BaseCommand):
+    help = "Associate hubs with subfields (Used for rep)"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--id",
+            default=None,
+            type=int,
+            help="Run for specific paper",
+        )
+
+        parser.add_argument(
+            "--start_id",
+            default=None,
+            type=int,
+            help="Paper start id",
+        )
+
+        parser.add_argument(
+            "--interacted_content_only",
+            default=False,
+            type=bool,
+            help="If used, only content that has been interacted with will be processed (e.g. papers associated with comments instaed of all papers)",
+        )
+
+    def handle(self, *args, **kwargs):
+        OA = OpenAlex()
+        papers = Paper.objects.all()
+
+        if kwargs["id"]:
+            papers = papers.filter(id=kwargs["id"])
+        elif kwargs["start_id"]:
+            papers = papers.filter(id__gte=kwargs["start_id"])
+
+        if kwargs["interacted_content_only"]:
```

Comment:
@koutst I think we don't need to retag every single paper in our DB. To speed things up, we can start with only papers that were interacted with (papers associated with comments + historically claimed)